boards/arm/ubx_evknorab10_nrf5340/Kconfig.defconfig

```diff
@@ -0,0 +1,87 @@
+# u-blox EVK-NORA-B10 board configuration
+
+# Copyright (c) 2021 u-blox AG
+# SPDX-License-Identifier: Apache-2.0
+
+if  BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP || BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP_NS
+
+config BOARD
+	default "ubx_evknorab10_nrf5340_cpuapp" if BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP || BOARD_UBX_EVKNORAB10_NRF5340_CPUAPP_NS
+
+# By default, if we build for a Non-Secure version of the board,
+# force building with TF-M as the Secure Execution Environment.
+
+config BUILD_WITH_TFM
+	# Temporarily disable building Non-Secure images with
+	# TF-M support by default. EVK-NORA-B10 is capable of
+	# TF-M, but not yet added to TrustedFirmware.org as a
+	# supported platform.
```

Comment:
Please remove TFM and non-secure related configurations until it can be used

Reply:
I'll get this added to TrustedFirmware.org soon. Pulled into another project ATM.